resources/leiningen/new/luminus/site/src/clj/oauth.clj

```diff
@@ -0,0 +1,44 @@
+(ns <<project-ns>>.oauth
```

Comment:
shouldn't be necessary to duplicate this, since it's already defined under `resources/leiningen/new/luminus/oauth/src/oauth.clj`